scss/progressCard.scss

```diff
@@ -0,0 +1,37 @@
+.progress-card__container {
+  display: flex;
+  border-radius: 8px;
+  box-shadow: 0 4px 10px 0 rgba(0, 0, 0, 0.07);
+  background-color: #5438dc;
+  color: #ffffff;
+  padding: 0.5rem;
+}
+
+.progress-card__progress-bar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 62px;
+  height: 62px;
+  border-radius: 50%;
+  border: solid 7.3px #3a22b8;
+  background-color: #5438dc;
+}
+
+.progress-card__title {
+  font-weight: bold;
+  line-height: 1.58;
+}
+
+.progress-card__paragraph {
```

Comment:
Can't you use Bootstrap to set this instead of creating your own css?

Reply:
Yeah I noticed this is default for bootstrap. Fixed 👍